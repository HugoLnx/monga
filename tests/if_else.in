int main(){
	if(2 == 0){
		a();
	}
	else{
		b();
	}
}