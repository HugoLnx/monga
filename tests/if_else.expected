program:
  function:main,259,0
    block:
      statements:
        statement:
          if:
            exp:7
              binary:5
                exp:0,2
                exp:0,0
            statement:
              block:
                statements:
                  statement:
                    function call:a
          else:
            statement:
              block:
                statements:
                  statement:
                    function call:b